ci/cloudbuild/triggers/clang-38-pr.yaml

```diff
@@ -5,7 +5,9 @@ github:
   pullRequest:
     branch: ^master$
     commentControl: COMMENTS_ENABLED_FOR_EXTERNAL_CONTRIBUTORS_ONLY
-name: clang-38
+name: clang-38-pr
 substitutions:
   _BUILD_NAME: clang-38
   _DISTRO: fedora
```

Comment:
Does fedora have Clang-3.8?

Reply:
Nope. That's a bug that I introduced in #6135. I'll fix that in a followup PR.